src/rviz/default_plugin/wrench_display.cpp

```diff
@@ -122,6 +122,12 @@ namespace rviz
 		       msg->header.frame_id.c_str(), qPrintable( fixed_frame_ ));
 	    return;
 	  }
+	
+	if ( position.isNaN() )
```

Comment:
Please make sure the indention is consistent with the rest of the file and uses spaces not tabs.